### PR TITLE
#US4R-424: Migrated RX section configuration to the us4r-api interface. 

### PR DESCRIPTION
**Pull request by pjarosik**

### CODE REVIEW

File: api/python/arrus/utils/imaging.py [hunk 1]
```diff
@@ -2819,7 +2819,7 @@ def prepare(self, const_metadata):
 
         self.tx_foc = self.num_pkg.asarray(focus, dtype=self.num_pkg.float32)
         burst_factor = tx_op.excitation.n_periods/(2 * self.fn)
-        self.initial_delay = -start_sample/65e6+burst_factor+tx_center_delay
+        self.initial_delay = -start_sample/120e6+burst_factor+tx_center_delay
```
Comment (jrozb91) [inline api/python/arrus/utils/imaging.py:2822]: Sampling frequency should depend on AFE(OEM) variant

File: api/python/arrus/utils/imaging.py [hunk 2]
```diff
@@ -3490,7 +3490,7 @@ def get_min_max_x_y(aperture):
         self.min_tang = self.num_pkg.float32(self.min_tang)
         self.max_tang = self.num_pkg.float32(self.max_tang)
         burst_factor = ref_tx.excitation.n_periods / (2 * self.fn)
-        self.initial_delay = -start_sample / 65e6 + burst_factor + tx_center_delay
+        self.initial_delay = -start_sample / 120e6 + burst_factor + tx_center_delay
```
Comment (jrozb91) [inline api/python/arrus/utils/imaging.py:3493]: Sampling frequency should depend on AFE(OEM) variant

File: api/python/arrus/utils/imaging.py [hunk 3]
```diff
@@ -3632,7 +3632,7 @@ def prepare(self, const_metadata):
 
         self.n_elements = probe_model.n_elements
         burst_factor = pulse.n_periods / (2 * self.fn)
-        self.initial_delay = -start_sample / 65e6 + burst_factor
+        self.initial_delay = -start_sample / 120e6 + burst_factor
```
Comment (jrozb91) [inline api/python/arrus/utils/imaging.py:3635]: Sampling frequency should depend on AFE(OEM) variant

File: api/python/arrus/utils/tests/utils.py [hunk 4]
```diff
@@ -175,8 +175,8 @@ def get_default_context(self, device=None, sequence=None, medium=None):
                     pitch=0.2e-3,
                     curvature_radius=0.0
                 ),
-                sampling_frequency=65e6,
-                data_sampling_frequency=65e6
+                sampling_frequency=120e6,
+                data_sampling_frequency=120e6
```
Comment (jrozb91) [inline api/python/arrus/utils/tests/utils.py:179]: Sampling frequency should depend on AFE(OEM) variant

File: arrus/core/devices/us4r/us4oem/Us4OEMDescriptorFactory.h [hunk 5]
```diff
@@ -77,6 +77,36 @@ class Us4OEMDescriptorFactory {
...
             };
+        case 3:
+            // us4OEM+ variant 0
+            return Us4OEMDescriptor{
+                version, // us4OEM version
+                32, // RX channels
+                20e-6f,  // min. RX time,
+                5e-6f, // RX time epsilon,
+                35e-6f, // TX parameters reprogramming time,
+                120e6f, // Sampling frequency [Hz]
```
Comment (jrozb91) [inline arrus/core/devices/us4r/us4oem/Us4OEMDescriptorFactory.h:88]: Sampling frequency should depend on AFE(OEM) variant

**Review by jrozb91 — CHANGES_REQUESTED** [review]

File: api/python/arrus/utils/imaging.py (hunk 1 above)
Comment (pjarosik) [reply api/python/arrus/utils/imaging.py]: Fixed

File: api/python/arrus/utils/imaging.py (hunk 3 above)
Comment (pjarosik) [reply api/python/arrus/utils/imaging.py]: Fixed

File: api/python/arrus/utils/tests/utils.py (hunk 4 above)
Comment (pjarosik) [reply api/python/arrus/utils/tests/utils.py]: I think this is acceptable, as this is an example (default) value used by the image reconstruction unit tests.

File: arrus/core/devices/us4r/us4oem/Us4OEMDescriptorFactory.h (hunk 5 above)
Comment (pjarosik) [reply arrus/core/devices/us4r/us4oem/Us4OEMDescriptorFactory.h:88]: This is the only place where the 120 MHz is right now stored -- as a property of the OEM HF descriptor. So this line is OK.

File: api/python/arrus/utils/imaging.py (hunk 2 above)
Comment (pjarosik) [reply api/python/arrus/utils/imaging.py]: Fixed

**Review by jrozb91 — APPROVED** [review]

**Pull request merged**